src/Tribe/Activation_Page.php

```diff
@@ -5,11 +5,41 @@
  */
 class Tribe__Events__Activation_Page {
 	/** @var self */
-	private static $instance = NULL;
+	private static $instance = null;
 
 	public function add_hooks() {
 		add_action( 'admin_init', array( $this, 'maybe_redirect' ), 10, 0 );
 		add_action( 'admin_menu', array( $this, 'register_page' ), 100, 0 ); // come in after the default page is registered
+
+		add_action( 'update_plugin_complete_actions', array( $this, 'update_complete_actions' ), 15, 2 );
+		add_action( 'update_bulk_plugins_complete_actions', array( $this, 'update_complete_actions' ), 15, 2 );
+	}
+
+	public function update_complete_actions( $actions, $plugin ) {
+		$plugins = esc_attr( $_GET['plugins'] );
+
+		if ( false !== strpos( $plugins, ',' ) ){
+			$plugins = explode( ',', $plugins );
+		}
+		$plugins = (array) $plugins;
+
+		if ( ! in_array( Tribe__Events__Main::instance()->pluginDir . 'the-events-calendar.php', $plugins ) ){
+			return $actions;
+		}
+
+		if ( isset( $actions['plugins_page'] ) ) {
+			$actions['plugins_page'] = '<a href="' . self_admin_url( 'plugins.php?tec-skip-welcome' ) . '" title="' . esc_attr__( 'Go to plugins page' ) . '" target="_parent">' . __( 'Return to Plugins page' ) . '</a>';
```

Comment:
Escaping:

* `esc_url()` the href.
* `esc_html__()` the Return to Plugins text